src/main/java/pl/tomaszdziurko/jvm_bloggers/blog_posts/data_migration/BlogPostUuidShorteningConvertingScheduler.java

```diff
@@ -0,0 +1,33 @@
+package pl.tomaszdziurko.jvm_bloggers.blog_posts.data_migration;
+
+
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.scheduling.annotation.Scheduled;
+import org.springframework.stereotype.Component;
+
+import pl.tomaszdziurko.jvm_bloggers.TimeConstants;
+
+@Component
+@Slf4j
+public class BlogPostUuidShorteningConvertingScheduler {
+
+    private static final int BATCH_SIZE = 250;
+
+    private final BlogPostUidShortener uuidShortener;
+
+    @Autowired
+    public BlogPostUuidShorteningConvertingScheduler(BlogPostUidShortener uuidShortener) {
```

Comment:
This handcrafted constructor can be replaced with class level annotation [`@RequiredArgsConstructor(onConstructor = @__(@Autowired))`](https://projectlombok.org/api/lombok/RequiredArgsConstructor.html).